README.md

```diff
@@ -64,9 +64,9 @@ $ make test
 
 ## Release
 
-Edit the VERSION file and change the new version. Submit a pull request to master. When it is merged, create a tag and push it to app-sre/sretoolbox.
+Update the VERSION file and change the new version. Submit a pull request to master. When it is merged, a CI job will be triggered that will publish the package on PyPI.
```

Comment:
I guess the CI job that publishes the package on PyPI is triggered after we `Draft a new release`. WDYT? :)

Reply:
The sequence is:
1. Update `VERSION` file
2. `git checkout main && git pull upstream main`
3. `git tag X.Y.Z`
4. `git push upstream X.Y.Z`
5. Draft an auto-generated release from the latest tag

Step 4 triggers a webhook to CI-EXT to execute `build_tag.sh` which creates a distributable reading the version from `VERSION` and pushes it to _PyPi_.

Step 5 is not strictly necessary.

Reply:
Hey @Ajpantuso, thanks for the reply and sorry for the late response. Am I reading this right that you don't have to commit anything before pushing a tag to upstream? And wouldn't we want a procedure that includes a merge request and a review?

Reply:
The `VERSION` file update requires a review/merge, but in general pushing a tag to trigger a release on any of the `mt-sre` repos is a one person operation. In principle we should review the commits introduced in a new release with the authors first to understand the impact, but requiring review of releases is risky (we would need to ensure two team members are available in all hotfix scenarios).

Reply:
As a side note this makes regression critical in CI and PR's that don't include tests should be heavily scrutinized. It all hinges on our discipline as contributers/reviewers.

Reply:
Ohhh I understand, step 1 is "Update VERSION file, create merge request, merge"